docs/content/direct/binding.md

```diff
@@ -1,5 +1,179 @@
 # Binding workload with WEC
 
-This document is about BindingPolicy and Binding objects and the concept of downsync.
+This document is about associating WECs with workload objects. The
+primary concept is sometimes called "downsync", which confusingly
```

Comment:
So why not just avoiding downsyncing in our docs and avoid 1) the confusion 2) this explanation about the confusion ?

Reply:
The term "downsync" is already all over the project. If you want to try eradicating it then I think that should be a separate PR.